integration-tests/it-common/src/main/java/fr/pilato/elasticsearch/crawler/fs/test/integration/elasticsearch/FsCrawlerTestFTPIT.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Licensed to David Pilato (the "Author") under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. Author licenses this
+ * file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package fr.pilato.elasticsearch.crawler.fs.test.integration.elasticsearch;
+
+import fr.pilato.elasticsearch.crawler.fs.client.ESSearchRequest;
+import fr.pilato.elasticsearch.crawler.fs.settings.Fs;
+import fr.pilato.elasticsearch.crawler.fs.settings.Server;
+import fr.pilato.elasticsearch.crawler.fs.settings.Server.PROTOCOL;
+import fr.pilato.elasticsearch.crawler.fs.test.integration.AbstractFsCrawlerITCase;
+import org.junit.Ignore;
+import org.junit.Test;
+
+/**
+ * Test crawler with FTP
+ * TODO: test framework is broken?
+ */
+public class FsCrawlerTestFTPIT extends AbstractFsCrawlerITCase {
+
+    /**
+     * You have to adapt this test to your own system
+     * So this test is disabled by default
+     */
+    @Test @Ignore
+    public void test_ftp() throws Exception {
+        String hostname = "192.168.18.207";
+        String username = "anonymous";
+        String password = "";
+
+        Fs fs = startCrawlerDefinition().build();
+        Server server = Server.builder()
+                .setHostname(hostname)
+                .setUsername(username)
+                .setPassword(password)
+                .setProtocol(Server.PROTOCOL.FTP)
+                .setPort(PROTOCOL.FTP_PORT)
+                .build();
+        startCrawler(getCrawlerName(), fs, endCrawlerDefinition(getCrawlerName()), server);
+
+        countTestHelper(new ESSearchRequest().withIndex(getCrawlerName()), 2L, null);
```

Comment:
I don't know why here is just not working(copied from `FsCrawlerTestSshIT.java`). Actually these tests don't need to have `@Ignore` annotation due to ftp fake server.